src/sentry/web/frontend/base.py

```diff
@@ -233,6 +236,25 @@ def get_team_list(self, user, organization):
             with_projects=True,
         )
 
+    def create_audit_entry(self, request, **kwargs):
+        if request.user.is_authenticated():
+            entry = AuditLogEntry.objects.create(
+                actor=request.user,
+                ip_address=request.META['REMOTE_ADDR'],
+                **kwargs
+            )
+            logger.info(
+                name='sentry.audit.entry',
+                event=entry.get_event_display(),
+                actor_id=request.user.id,
+                actor_label=entry.actor_label,
+            )
+
+        else:
+            logger.warn('Attempted to audit event for an AnonymousUser at %s' %
```

Comment:
We can use `AuditLogEntry` without an authenticated user. The `actor_id` column is nullable. So you could just do:

```python
actor=request.user if request.user.is_authenticated() else None
```